### ISSUE TITLE
Searching by group default

### ISSUE THREAD

**Issue by ecurrano** [paleobot/pbot-dev#292]
We should discuss whether we want to maintain the default of only searching public when a search is run and no group is selected. Ideally, I would like a search of public plus all my groups. This could be handled through training if it is difficult to implement. And, by having to manually select public plus all your private groups, it does encourage people to make their data public.

If we are going to make people select groups, then perhaps we should move this near to the top of the search page so that people (i.e., Ellen) don't forget to do it.

**Comment by aazaff** [comment]
It does sound like a good idea to make the default be whatever the user is authorized to see (i.e., public + groups), but it may take more time than we currently have.

**Comment by doricon** [comment]
I agree - prefer to default search by everything I am authorized to see.

**Comment by doricon** [comment]
Revisiting - I think that moving the group/public selection is a simple solution that would be very functional. I agree it is not very easy to remember to do when hidden in the metadata accordion. I think of a search process as first choose which datasets you want to search (ie the group selection), then use the accordions to choose your specific criteria for the search. 

I think this would pertain to the Taxon, Collection, and Specimen searches. Place the group drop down right under "search parameters" before Name.

**Comment by ecurrano** [comment]
I am bumping this back to attention because I think it is a very high priority to be more easily able to search in private groups. If it is too difficult to have a default search be both public and all private groups, then I advocate to move Groups in all search menus to the top of the list and outside of an accordion.

**Comment by ecurrano** [comment]
I am bumping this back up again. I understand that it is too much work to be able to change the default to search based on everything that a user has access to. This is a v2 feature! My request is that search by Group be moved out of the metadata dropdown (PBot Explore: OTU, Collection, Specimen) and placed immediately below Name or Specimen number and above all the accordions. 

It would also be nice to have this for the search menus in the Workbench for editing OTUs, Collections, and Specimens, but that is lower priority than the main Explore functionality.

**Comment by ecurrano** [comment]
I have Andrew here giving me permission to change this priority. The request is that in the various searches, we have a group dropdown right at the top so that you do not need to go searching in Metadata for it. @NoisyFlowers
